### DISCUSSION TITLE
Tool could remember last secret used

### DISCUSSION THREAD

**Discussion by wizofaus** [CorentinTh/it-tools#1208] [Ideas]
I'd used the OTP tool a few times, it had worked great (thanks!) but then after my browser had restarted, the OTP codes stopped working - because it had seemingly chosen some new random secret. I would have thought it could remember what I'd used last time...(or at worst default to blank and require putting one in again).

**Comment by sharevb** [comment]
Hi @wizofaus, probably better to open an issue : https://github.com/CorentinTh/it-tools/issues/new/choose

**Comment by sharevb** [comment]
Hi @wizofaus, implemented in #1221

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
